backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/DefaultGwtInput.java

```diff
@@ -1223,6 +1233,11 @@ protected int keyForCode (int keyCode, int location) {
 	private static final int KEY_F24 = 135;
 	private static final int KEY_NUM_LOCK = 144;
 	private static final int KEY_SCROLL_LOCK = 145;
+	private static final int KEY_AUDIO_VOLUME_DOWN = 174;
```

Comment:
https://keycode.info/ states this is 182 on Firefox, 183 for Volume up

Reply:
Good find, will add it